### DISCUSSION TITLE
have any principles for handling punctuation?

### DISCUSSION THREAD

**Discussion by yzmyyff** [lhotse-speech/lhotse#1223] [General]
I noticed that we are now getting rid of all punctuation in recipes. 
However, punctuation can still be useful in certain tasks.
Can I control punctuation removal during data preparation?

**Comment by desh2608** [comment]
Most recipes provide an optional `--normalize-text` option to control whether you want to normalize the supervisions in a certain way. If you don't see it as an option in the recipe you are using, please feel free to open a PR.
